src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -671,13 +671,30 @@ export function getAppointmentSlots(startDate, endDate, forceFetch = false) {
   };
 }
 
-export function onCalendarChange(selectedDates) {
+export function onCalendarChange(
```

Comment:
Changed to check for duplicate appointments in the action since the continue button is in the parent component.